src/Psy/ExecutionLoop/Loop.php

```diff
@@ -21,7 +21,7 @@
 class Loop
 {
     /**
-     * The non-forking loop doesn't have much use for Configuration :)
+     * The non-forking loop doesn't have much use for Configuration :).
```

Comment:
I only replaced as csfixer asked, should we open a issue that the smileys should be used as a punctuation chars?